agent/js/src/wd_server.js

```diff
@@ -1028,13 +1028,34 @@ WebDriverServer.prototype.onTracingMessage_ = function(message) {
 };
 
 /**
- * Runs lighthouse with the options specified by the task and dumps data into 
+ * Runs lighthouse with the options specified by the task and dumps data into
  * a directory.
  * @private
  */
 WebDriverServer.prototype.runLighthouse_ = function() {
-  // TODO(phulce): actually run lighthouse
-  this.onPageLoad_(new Error('Lighthouse not yet implemented!'));
+  this.lighthouseFile_ = path.join(this.runTempDir_, 'lighthouse.html');
+  this.app_.timeout(1000, 'Waiting for Chrome to be available...');
+  process_utils.scheduleExec(this.app_,
+    'lighthouse',

```

Comment:
How big is it and is it feasible to include directly?  Worst case I'll add logic to the agent at startup to look for it and try to install if it isn't there (and automatically update if it is) but if we can eliminate the need for manually installing then all of the existing agents will pick up lighthouse support automatically.

Also, does running it as "lighthouse" work on Windows as well or does Node need to explicitly be referenced?  Just checking since the Node agent runs across all platforms.